StardewArchipelago/Locations/CodeInjections/Modded/SVE/BearShopStockModifier.cs

```diff
@@ -52,7 +52,7 @@ private void MakeBearBarter(ShopData shopData)
             foreach (var shopItem in shopData.Items)
             {
                 var isRecipe = shopItem.ItemId.Contains("Baked Berry Oatmeal") || shopItem.ItemId.Contains("Flower Cookie");
-                if (!_archipelago.SlotData.Chefsanity.HasFlag(Chefsanity.Purchases) || !isRecipe)
+                if (!_archipelago.SlotData.Chefsanity.HasFlag(Chefsanity.Purchases) || _archipelago.SlotData.Cooksanity == Cooksanity.All || !isRecipe)

```

Comment:
The original change was because if someone needed to buy the recipe for a check (due to chefsanity), it should be the easily accessible berries.  Someone complained it was too hard on *cooksanity*, where you still need the recipe, but to cook it.  So I made the same rule for cooksanity.